<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VirtualRobotBottomWidget</class>
 <widget class="QWidget" name="VirtualRobotBottomWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>66</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QToolButton" name="stopBtn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>56</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>56</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;
background-color:transparent;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
     <property name="icon">
      <iconset>
       <normalon>:/image/res/image/virtualRobotBottom/ic_stop.png</normalon>
       <disabledon>:/image/res/image/virtualRobotBottom/ic_stop_disable.png</disabledon>
      </iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="playBtn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>56</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;
background-color:transparent;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Play</string>
     </property>
     <property name="icon">
      <iconset>
       <normalon>:/image/res/image/virtualRobotBottom/ic_play.png</normalon>
       <disabledon>:/image/res/image/virtualRobotBottom/ic_play_disable.png</disabledon>
      </iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="modifyBtn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>56</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;
background-color:transparent;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Modify</string>
     </property>
     <property name="icon">
      <iconset>
       <normalon>:/image/res/image/virtualRobotBottom/ic_change.png</normalon>
       <disabledon>:/image/res/image/virtualRobotBottom/ic_change_disable.png</disabledon>
      </iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="resetBtn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>56</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;
background-color:transparent;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Reset</string>
     </property>
     <property name="icon">
      <iconset>
       <normalon>:/image/res/image/virtualRobotBottom/ic_reset.png</normalon>
       <disabledon>:/image/res/image/virtualRobotBottom/ic_reset_disable.png</disabledon>
      </iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="addBtn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>56</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;
background-color:transparent;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
     <property name="icon">
      <iconset>
       <normalon>:/image/res/image/virtualRobotBottom/ic_add.png</normalon>
       <disabledon>:/image/res/image/virtualRobotBottom/ic_add_disable.png</disabledon>
      </iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
